winui/Shadow/Overview.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: What's New for .NET WinUI Shadow Control | Syncfusion
+description: The .NET WinUI Shadow control is used to apply shadow effects to any framework element to create a beautiful and appealing UI.
+platform:winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# WinUI Shadow (New)
```

Comment:
@Priyanka-Vijayasankar WinUI Shadow Overview.

Reply:
Removed new word in WinUI Shadow Overview title.